### ISSUE TITLE
Not Working Features

### ISSUE THREAD

**Issue by arjay07** [zhuowei/MCPELauncher#1497]
Is there an updated list of features that is currently not working on BlockLauncher?

**Comment by Haha124** [comment]
No, I think there is currently not a list, but I think only the custom things like Textures, Blocks and Items don't work.
